script/bundle-linux

```diff
@@ -38,27 +38,31 @@ version_info=$(rustc --version --verbose)
 host_line=$(echo "$version_info" | grep host)
 target_triple=${host_line#*: }
 musl_triple=${target_triple%-gnu}-musl
+remote_server_triple=${REMOTE_SERVER_TARGET:-"${musl_triple}"}
 
 # Generate the licenses first, so they can be baked into the binaries
 script/generate-licenses
-rustup target add "$musl_triple"
+
+if [[ "$remote_server_triple" == "$musl_triple" ]]; then
+    rustup target add "$musl_triple"
```

Comment:
Thinking of the generic approach to this, does it make sense to install `musl_triple` at all?
Seems that we'd better add `remote_server_triple` unconditionally?

Reply:
Probably better here, doing next: try to add target by rustup only if remote_server_triple different from host rust compiler (default target). 
If $remote_server_triple the same what `rustc --version --verbose` return is have no sens to change anything.

Reply:
Yeah, that also could work.

My current concern is that anything can be put into `REMOTE_SERVER_TARGET` "argument", and if that is not installed along the way, we'll get more cryptic messages.

Not insisting strongly, lmk if you want to keep things as is and merge.